queries/php_only/highlights.scm

```diff
@@ -1,3 +1,168 @@
+; Keywords
+[
+  "and"
+  "as"
+  "instanceof"
+  "or"
+  "xor"
+] @keyword.operator
+
+[
+  "fn"
+  "function"
+] @keyword.function
+
+[
+  "break"
+  "class"
+  "clone"
+  "declare"
+  "default"
+  "echo"
+  "enddeclare"
+  "enum"
+  "extends"
+  "global"
+  "goto"
+  "implements"
+  "insteadof"
+  "interface"
+  "print"
+  "namespace"
+  "new"
+  "trait"
+  "unset"
+] @keyword
+
+[
+  (abstract_modifier)
+  "const"
+  (final_modifier)
+  (visibility_modifier)
+  (readonly_modifier)
+  (static_modifier)
```

Comment:
I still think capturing keywords is better than its parent

Reply:
How is it better?

Reply:
It's still more specific. And for neovim, priority > specificity > precedence. Because yours isn't fully specific, those which override queries may have weird highlights

Reply:
Alrighty, reverted